components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/backchannellogout/DefaultLogoutTokenBuilder.java

```diff
@@ -161,15 +161,15 @@ private JWTClaimsSet buildJwtToken(OIDCSessionState sessionState, String tenantD
         long currentTimeInMillis = Calendar.getInstance().getTimeInMillis();
         Date iat = new Date(currentTimeInMillis);
         String sid = getSidClaim(sessionState);
-        JSONObject event = new JSONObject().put("http://schemas.openidnet/event/backchannel-logout",
+        JSONObject event = new JSONObject().appendField("http://schemas.openidnet/event/backchannel-logout",
```

Comment:
Lets add a constant for this

Reply:
Fixed in - 09e790c015d97acdf73780ab3a4af4e027c70a78, 667595f9efca0c7534ab23e4eabb407bbd6697f5